src/js/api/ajax.js

```diff
@@ -0,0 +1,19 @@
+const { YOUTUBE_BASE_URL } = process.env;
+const { YOUTUBE_API_KEY } = process.env;
+
```

Comment:
여기는 이렇게 줄일 수 있을 것 같아요ㅎㅎ 
```suggestion
const { YOUTUBE_BASE_URL, YOUTUBE_API_KEY } = process.env;
```

Reply:
ㅠㅠ이게 .env 문제인지 모르겠는데 일반 js객체가 아닌지 참조가 안되더라고요... 특이하게 저렇게 따로 참조하면 읽히는... 하하...